CHANGELOG.latest.md

```diff
@@ -1,2 +1,7 @@
-- Introduced New AdServices Integration on iOS. Call `EnableAdServicesAttributionTokenCollection()` to enable this integration.
-More information [in our docs](https://docs.revenuecat.com/docs/apple-search-ads) and in the [announcement blogpost](https://www.revenuecat.com/blog/iad-vs-adservices-whats-the-difference/).
+### API Changes
+* `StoreTransaction`: `RevenueCatId` and `ProductId` have been deprecated in favor of `TransactionIdentifier` and `ProductIdentifier` respectively. (#145) via Toni Rico (@tonidero)
+### Bugfixes
+* Fix example compatibility with Unity 2020 (#139) via Andy Boedo (@aboedo)
+### Other Changes
+* Subtester: Fix unity android export (#142) via Andy Boedo (@aboedo)
+* Update AMAZON-INSTRUCTIONS.md (#143) via Andy Boedo (@aboedo)
```

Comment:
can we maybe add a big callout warning folks using Unity IAP side-by-side with RevenueCat to ensure they update to 4.4.0+ if they're using this version?

Reply:
Good callout! And done!